### ISSUE TITLE
Not named .groovy

### ISSUE THREAD

**Issue by MikeBishop** [robertvandervoort/SmartThings#24]
Since the files aren't named .groovy, SmartThings' GitHub integration doesn't see them.  You might want to consider moving to .groovy filenames and using release tags to mark your versions instead of having distinct files.

**Comment by robertvandervoort** [comment]
I have actually. The first time I used GitHub was because of SmartThings. I
had no idea what I was doing LOL!

I am going to be moving all my device handler code to separate repos all
with proper naming and cleaning out old releases.

Thanks for the feedback!

**Comment by rhproulx** [comment]
Hi. Appreciate what you do! I am a rooky trying to use your code with Smartthings to run Aeon Labs Smart dimmer 6 and Micro Double smart switch. I am confused. Is a Device handler what I would use for Smarthings App? I can get the dimmer installed but it won't operate. Does it need a Smartapp? Help!
Thanks.

**Comment by apwelsh** [comment]
Your devices need both.  The device handler is the integration code that allows smart things to talk to the device, and listen to what the device has to say.  The Smarthings Apps are the automation components that tie different devices together, or perform actions based on events.  The Smart Dimmer 6 and the Micro Double Smart Switch are load controlling devices.  This means that a device handler is all that is needed to control these devices using the SmartThings app.  If, however, you want to have a wall switch, morion sensor, proximity sensor, time of day, or some other trigger cause the dimmer and switch to change state, then you need the Smart App to create the automation rules.

**Comment by robertvandervoort** [comment]
correct. Also, I posted a newer version of the smart dimmer code, you'll
want to use that instead.

**Comment by rhproulx** [comment]
Thanks Guys! I'll try it tommorrow.

*Roger H Proulx II*
*Retired General Contractor*
*VPSHOA Board Member*
*15152C Campus Park Drive*
*Moorpark, CA 93021*
*805-517-1905*



On Thu, Nov 9, 2017 at 1:59 PM, Robert Vandervoort <notifications@github.com

**Comment by rhproulx** [comment]
I have not had time to apply this advice until today and it worked
perfectly for the Smart Dimmer 6. Next I will try to install the Micro
Double switch . Will keep you posted!

Thanks So Much!

*Roger H Proulx II*
*Retired General Contractor*
*VPSHOA Board Vice President*
*15152C Campus Park Drive*
*Moorpark, CA 93021*
*805-517-1905*
